hs_core/tasks.py

```diff
@@ -71,7 +71,7 @@ def __init__(self, error_message):
         super(FileOverrideException, self).__init__(self, error_message)
 
 
-@celery_app.on_after_configure.connect
+@celery_app.on_after_finalize.connect
```

Comment:
So the celery_app tasks aren't ready to be used with a period_task during on_after_configure (at least some of the tasks)?  Not a ton of information on how this works in the documentation but this seems to be a reasonable approach.  https://docs.celeryq.dev/en/stable/reference/celery.html#celery.Celery.on_after_configure

Reply:
not a lot of info but I found this thread helpful
https://github.com/celery/celery/issues/3589#issuecomment-290979095
and making the change on beta seems to kick things into gear